### ISSUE TITLE
Google App Engine (GAE) - Flexible Environment - Jetty Issue

### ISSUE THREAD

**Issue by washowasho** [GoogleCloudPlatform/appengine-maven-plugin#86]
I also created this issue on StackOverflow: https://stackoverflow.com/questions/44551221/google-app-engine-gae-flexible-environment-jetty-issue

My first question is, what's the difference between this plugin and the com.google.cloud.tools - appengine-maven-plugin? Is this project for Java 7 / standard, and the com.google.cloud.tools one for Java 8?

Next question is as follows:

After using mvn appengine:deploy, we receive a BUILD SUCCESSFUL message... but when you navigate to the URL... the following message is received:

```
Error 404 - Not Found.

No context on this server matched or handled this request.
Contexts known to this server are:
/ ---> o.e.j.w.WebAppContext@4566e5bd{/,file:///var/lib/jetty/webapps/root/,UNAVAILABLE}{/root.war} [failed]

Powered by Jetty:// 9.4.5.v20170502
```

I almost feel like the WAR is being deployed to the wrong directory on the AppEngine, causing Jetty to not be able to locate. Any ideas?

**Comment by washowasho** [comment]
This can be marked as closed. I was able to get it deployed by rebuilding my project. I think there were certain JAR files present that AppEngine did not like.

**Comment by karikalankumaresan** [comment]
Hi, I get the same error when I try to access the url of the app deployed in appengine (flex). I have few questions: 
1. Why I get an error about jetty while I have configured to run tomcat. 
2. When I used jar packaging, it worked fine (but was not processing jsp pages), so I changed it to war packaging (with webapp/WEB-INF/jsp/<jsp files>), it works fine in local, however not so in appengine, I get the above error you mentioned.
3. I do not have the web.xml for the webapp defined, do I need to do that? The tutorial I followed did not mention anything about it.

Any ideas?
